StantDrawing/BrushTool/BrushToolView.swift

```diff
@@ -187,27 +221,115 @@ public class BrushToolView: UIView {
+        
+        pencilIconView.addSubview(pencilIconImageView)
 
-        if let uiButton = groupPencilSizeButtons.first?.uiButton {
-            setBtnSizeAsClicked(button: uiButton)
-        }
+        // Separator
+        let separatorView   = UIView()
+        let originSeparator = CGPoint(x: 10, y: pencilIconView.frame.origin.y + pencilIconView.frame.height + 10)
+        let sizeSeparator   = CGSize(width: groupViewWidth - 20, height: 1)
+        let frameSeparator  = CGRect.init(origin: originSeparator, size: sizeSeparator)
+
+        separatorView.frame = frameSeparator
+        separatorView.backgroundColor = UIColor.black
+        separatorView.alpha = 0.2
+
+        // Group Pencil Buttons
+        let groupPencilBtns = UIView()
+        let origin          = CGPoint(x: 0, y: separatorView.frame.origin.y + separatorView.frame.height + 10)
+        
+        groupPencilBtns.frame.origin = origin
+        
+        configureGroupLayout(buttons: groupPencilSizeButtons, groupView: groupPencilBtns, groupViewOirigin: origin, addAsSubview: false)
+        
+        setAllGroupdPencilSizeButtonsAsNotClicked()
+        
+        setBtnSizeAsClicked(button: groupPencilSizeButtons[1].uiButton)
+        
+        // GroupPencilSizeView Configs
+        let height    = pencilIconView.frame.size.height + separatorView.frame.size.height + groupPencilBtns.frame.size.height
+        let sizeGroup = CGSize(width: self.frame.width, height: height)
+        groupPencilSizeView.frame.size = sizeGroup
+        groupPencilSizeView.isHidden   = true
+        
+        groupPencilSizeView.addSubview(pencilIconView)
+        groupPencilSizeView.addSubview(separatorView)
+        groupPencilSizeView.addSubview(groupPencilBtns)
     }
     
     private func configureGroupEraseSizeLayout() {
-        let origin = CGPoint(x: groupToolsView.frame.width + groupToolsView.frame.origin.x + 10, y: 0)
-        configureGroupLayout(buttons: groupEraseSizeButtons, groupView: groupEraseSizeView, groupViewOirigin: origin, addAsSubview: false)
-        groupEraseSizeView.isHidden = true
+        // Erase Icon
+        let eraseIconView   = UIView()
+        
+        let originIconView = CGPoint(x: 0, y: 0)
+        let sizeIconView   = CGSize(width: groupViewWidth, height: 60)
+        let frameIconView  = CGRect.init(origin: originIconView, size: sizeIconView)
+        
+        eraseIconView.frame              = frameIconView
+        eraseIconView.backgroundColor    = UIColor(hex: "F0F0F0")
+        eraseIconView.layer.cornerRadius = eraseIconView.frame.width / 2
+        
+        let originIconImg = CGPoint(x: (groupViewWidth - 25) / 2, y: 18)
+        let sizeIconImg   = CGSize(width: 25, height: 25)
+        let frameIconImg  = CGRect.init(origin: originIconImg, size: sizeIconImg)
+        
+        let eraseIconImageView = UIImageView(image: UIImage(named: "eraserfull", in: bundle, compatibleWith: nil))
```

Comment:
Same comment as stated before about images extensions.

Reply:
I don't believe in this case is necessary. We have only 1 line using it.